lib/smart_proxy_container_gateway/container_gateway_api.rb

```diff
@@ -120,6 +146,8 @@ class Api < ::Sinatra::Base
             ContainerGateway.update_user_repositories(request.params['account'],
                                                       JSON.parse(repo_response.body)['repositories'])
           end
+
+          # Return the original token response from Katello
           return token_response_body.to_json
```

Comment:
Should this make it compliant with the spec and drop the `expires_at` header? If not, you can simply return the original string, saving the cost of encoding again.
```suggestion
          # Return the original token response from Katello
          return token_response.body
```

Reply:
+1 to this

Reply:
Fixed, good catch.